climbing-stairs/Leo.py

```diff
@@ -0,0 +1,12 @@
+class Solution:
+    def climbStairs(self, n: int) -> int:
+        fast, slow = 1, 1
+
+        for i in range(n - 1):
+            tmp = fast
+            fast = fast + slow
+            slow = tmp
```

Comment:
파이썬스럽게 작성한다면 이렇게도 해볼 수 있을 것 같아요.

```suggestion
            fast, slow = fast + slow, fast
```